Tests/MeiliSearchUnitTests/DocumentsTests.swift

```diff
@@ -6,10 +6,10 @@ import Foundation
 // swiftlint:disable force_try
 // swiftlint:disable line_length
 private struct Movie: Codable, Equatable {
-  let id: Int
-  let title: String
-  let overview: String
-  let releaseDate: Date
+  let id: Int?
+  let title: String?
+  let overview: String?
+  let releaseDate: Date?
```

Comment:
overview and releaseDate seem to be optional in some of your tests, but `id` and `title` are used everywhere. Do they have to be optional ?